modules/contrib/src/components/Renderable.ts

```diff
@@ -85,6 +84,6 @@ export class Renderable<T extends RenderableProps = RenderableProps> implements
     this.c.removeAllListeners();
 
     // remove from the world
-    this.c.destroy();
+    // this.c.destroy(); // TODO disabled because it breaks from collider debug
```

Comment:
this line crashed the game (when turning off debug mode) with `this._geometry is null` from the pixi `Graphics` class

don't know root cause